scorecard/README.md

```diff
@@ -23,9 +23,20 @@ The Relax scorecard is intended benchmark Relax against other frameworks. The mo
 
 ### One-Time Setup
 
+#### AWS Instance Setup
+
+If you intend to run on an AWS instance this may help replicate the scorecard results. If you already have a machine with the appropriate CUDA driver and GPUs you can skip this step.
+1. Go to the [AWS EC2 "Launch Instance" page](https://us-west-2.console.aws.amazon.com/ec2/home?region=us-west-2#LaunchInstanceFromTemplate:launchTemplateId=lt-0b2796d6ff89f5313) for `octoml-sandbox` as a `Sandbox-Developer`.
+2. Select your AMI. The recommended AMI is `gitlab-gpu-runner-v202302190039` (id: `ami-0cca4cca5ab8eb905`)
```

Comment:
don't need steps 2, 3, or 5 anymore, also maybe we could update the text above to say "launch using the `test-mlsys-dev` launch template" or something?